test/testscript.py

```diff
@@ -0,0 +1,6 @@
+import pymongo
+import sys
+print(sys.argv)
+client = pymongo.MongoClient(serverSelectionTimeoutMS=1000)
```

Comment:
Please also add a docstring to this script that mentions that its purpose is to test the CLI functionality of pymongoexplain.

Reply:
Done.